### ISSUE TITLE
Share sheets between players

### ISSUE THREAD

**Issue by Saborknight** [Saborknight/9liner-and-notepad#3]
Allow sharing of sheets (through ACE? Like the "Copy Map" function).

---

**Source**: [BI Forums](https://forums.bistudio.com/forums/topic/178540-ingame-9liners-notepad/?do=findComment&comment=2807610)

> But actually I'd be fine without inventory items for this mod. A copy-function (like the copy map-function from ACE) would be enough - it'd be great though if you could copy off of dead player's bodies just as well as from alive ones.

**Submitted by** [Belbo](https://forums.bistudio.com/profile/828096-belbo/)

**Comment by TheMysteriousVincent** [comment]
If we (as in #9 mentioned) have added a slightly more dynamic system, we can move sheets to one another very easily.

**Comment by Saborknight** [comment]
This feature is more related to sharing between different players rather than just copying to another sheet in your own notepad. However, that in itself would also be quite a good feature to have... See #12 for continuation on that front.

**Comment by TheMysteriousVincent** [comment]
Just for clarification: I meant the sharing between players.

**Comment by Saborknight** [comment]
Neeevermind then, I'd say we're on the same page and on track then xD

**Comment by TheMysteriousVincent** [comment]
Any suggestions on how to select a single page and give it to another player? Like the UI the player has to use, e.g.

**Comment by Saborknight** [comment]
Well, obviously it needs to be ACE independent.

For when ACE doesn't exist: use the main menu Dialogue listing sheet formats. Click through to a dialogue giving page numbers appended with first X number of characters followed by Ellipsis, and has interaction options (such as add sheet, delete sheet, duplicate, share, etc) next to each format."Share" clicks through to a list of nearby players. Don't forget back and "X" buttons

For when ACE exists: interaction on another person, offer the option under `Sheets >> Share >> [name of sheet format] >> Page xx`.

That's how I see it should be. Comments?

**Comment by TheMysteriousVincent** [comment]
> For when ACE exists: interaction on another person, offer the option under Sheets >> Share >> [name of sheet format] >> Page xx.

I don't think that this is a good idea because that would be too messed up in the end. Imagine if players have more than 30 pages on their hand and they have to select a single page. To select a single page would be pretty different.

Besides that, I think the menu is a good idea. But shall this menu be an overall menu to manage all pages? I mean this isn't anything bad but that would be a duplicate feature.

**Comment by Saborknight** [comment]
You're right, the menu thing I was talking about is a separate feature which has a "Share" option for control of this feature.

**Comment by Saborknight** [comment]
Here's an example of a similar venture done by Salbei

**Comment by pixlcrashr** [comment]
Which example?

**Comment by Saborknight** [comment]
Apologies, I was half asleep and obviously forgot to include the link: https://github.com/gruppe-adler/grad-leaveNotes
